modules/custom-tuning-specification.adoc

```diff
@@ -134,12 +134,11 @@ ifdef::rosa-hcp-tuning[]
   "recommend": [
     {
       "profile": <tuned_profile_name>, <1>
```

Comment:
All of these changes to remove references to machineConfigLabels only apply to HCP, so they should be copied to a separate section behind an ifdef. We don't want to change these docs in the standalone OCP docs.

Reply:
Checked preview of topic in OCP with current (published) topic in OCP and they match, so conditionals have separated the changes (i.e. current OCP docs will not be affected by changes).